lib/generators/mobile_workflow/install/templates/config/initializers/mobile_workflow_rollbar.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module MobileWorkflow
+  class ApplicationJob < ActiveJob::Base
+    include Rollbar::ActiveJob
+  end
+end
```

Comment:
I'm not sure about this. If after generating the app someone creates an `app/jobs/application_job.rb` file with custom logic, will this initializer override the custom one or the opposite? shouldn't be better to just add this file you created to `mobile_workflow/lib/generators/mobile_workflow/install/templates/app/jobs/application_job.rb` ?

Reply:
But if someone creates an `app/jobs/application_job.rb`, they will define an `ApplicationJob` that is not scoped under `MobileWorkflow`, and the above file is only overriding the one defined in the gem. Same if I set it as a template under `mobile_workflow/lib/generators/mobile_workflow/install/templates/app/jobs/application_job.rb`; it will not be scoped by `MobileWorkflow`.

Reply:
For me it's a bit confusing to define a class inside an initializer. Also, when creating the first active job you will need to remember that you need to inherit from this `MobileWorkflow:: ApplicationJob`. If you add it in `app/jobs/application_job.rb` there is no risk to forget it.

Reply:
Good point! I thought we would add it manually if the project has application_jobs, but I am sure it will do in most of the cases, so I'll add it too. Thanks!